code/headers/test_usart.hpp

```diff
@@ -2,20 +2,21 @@
 #include <usart_connection.hpp>
 #include <uart_ports.hpp>
 #include <queue.hpp>
-#include <hwlib.hpp>
+#include <vector>
+#include <string>
 
 namespace r2d2 {
 
     class test_usart_c : public usart_connection_c {
     private:
-        unsigned int boudrate;
+        unsigned int baudrate;
         uart_ports_c usart_port;
-        queue_c<uint8_t, 250> input_buffer;
+        queue_c<uint8_t, 250> receive_buffer;
 
 
     public:
 
-        test_usart_c(unsigned int baudrate, uart_ports_c usart_port);
+        test_usart_c();
 
         /// @brief does not actualy disable anything
```

Comment:
this comment is wrong btw

Reply:
edited in https://github.com/R2D2-2019/USART_library/pull/12/commits/85fdce96b4074228fcd84ff4c1db1e03da94d3f2